interactions/api/models/guild.py

```diff
@@ -2019,6 +2020,157 @@ async def delete_emoji(
             if int(item.id) == int(emoji_id):
                 return self.emojis.remove(item)
 
+    async def get_stickers(self) -> List[Sticker]:
+        """
+        Get the stickers for a guild.
+
+        :return: List of stickers of the guild.
+        :rtype: List[Sticker]
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
+        res = await self._client.list_guild_stickers(guild_id=int(self.id))
+
+        stickers = [Sticker(**sticker) for sticker in res]
+        self.stickers = stickers
+
+        return self.stickers
+
+    async def get_nitro_sticker_packs(self) -> List[StickerPack]:
+        """
+        Gets the list of sticker packs available to Nitro subscribers.
+
+        :return: List of sticker packs.
+        :rtype: List[StickerPack]
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
+        res = await self._client.list_nitro_sticker_packs()
+
+        return [StickerPack(**sticker_pack) for sticker_pack in res["sticker_packs"]]
+
+    async def create_sticker(
+        self,
+        file: File,
+        tags: str,
+        name: Optional[str] = MISSING,
+        description: Optional[str] = MISSING,
+        reason: Optional[str] = None,
+    ) -> Sticker:
+        """
+        Creates a new sticker for the guild.
+
+        :param file: The file of the sticker.
+        :type file: File
+        :param tags: The tags of the sticker.
+        :type tags: Optional[str]
+        :param name: The name of the sticker.
+        :type name: Optional[str]
+        :param description: The description of the sticker.
+        :type description: Optional[str]
+        :param reason: The reason of the creation.
+        :type reason: Optional[str]
+        :return: Created sticker for the guild.
+        :rtype: Sticker
+        """
+        if not self._client:
+            raise LibraryException(code=13)
+
```

Comment:
Iirc discord has strict limitations for stickers, make sure they are met before doing the request, otherwise raise lib exc

Reply:
I spammed create_sticker and i haven't been ratelimited for this. Only for delete_sticker

Reply:
```
The HTTP client has encountered a per-route ratelimit. Locking down future requests for 0.0 seconds.
...
RuntimeError: Form data has been processed already
...
TypeError: interactions.api.models.message.Sticker() argument after ** must be a mapping, not NoneType
```
This?

Reply:
I thought about file type limits and stuff but it isn't that important ig